tests/Exceptions/ValidationExceptionTest.php

```diff
@@ -98,4 +111,3 @@ public function providerForFormat()
         );
     }
 }
-
```

Comment:
Two newlines were at end of file. Github doesn't show the other one, but it is there if you clone the repo.